src/peer/api/api-local-proxy.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Local API proxy - same process
+ */
+class ApiLocalProxy {
+  constructor(apiHandler) {
+    this.apiHandler = apiHandler;
+  }
+
+  /**
+   * List rooms from the local API instance
+   * @return {Promise<any|*[]>}
+   */
+  async listRooms() {
+    return this.apiHandler.listRooms();
```

Comment:
Missing `await` or doesn't need `async`

Reply:
`await` it's not needed here. It should stay `async` though since I wanted to be the same signature ass the method used in `api-remote-proxy`